src/connector/with_options_sink.yaml

```diff
@@ -413,6 +413,12 @@ NatsConfig:
   - name: subject
     field_type: String
     required: true
+  - name: filter_subject
+    field_type: String
+    required: false
+  - name: durable
+    field_type: bool
+    required: false
```

Comment:
Hey, I guess `durable` is only for source, right?

If so, you may only add it to NatsSourceConfig instead of NatsCommon.